src/ReflectionClosure.php

```diff
@@ -626,6 +626,10 @@ protected function getHashedFileName()
      */
     protected function getFileTokens()
     {
+        if (stripos($this->getFileName(), "eval()'d code") > 0) {
```

Comment:
Usual to do `!== false` I think?

Reply:
In this case, if the filename contains this string the position will most likely be greater than 0 , in my use case the string position was well over 50 characters.  so !== false would not work.
For the normal use case the string index position will be -1  (negative 1)

Reply:
resolved

Reply:
I will test it with !== false and see what happens.
Ok, yes my tests succeed also with !== false, I believe this covers it, if the string index is 0 it should not be false and still pass , otherwise -1 is the normal case.